### DISCUSSION TITLE
Welcome to akasha-world-framework Discussions!

### DISCUSSION THREAD

**Discussion by kenshyx** [AKASHAorg/akasha-core#955] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by felixboehm** [comment]
Fullstack and web3 developer, former working in file cloud, open source company ownCloud. 
I am thrilled about ethereum.world and want to participate, contribute and learn from the project.
Discovered the project during research on similar projects to own ideas and super happy to not start from scratch. 
I want to help enabling community contributions, test automation and CI/CD pipelines, spreading the word, and architect, design, develop an extension system as an early adopter writing plugins or integrations. 

Thanks everyone for building this!

**Reply by josenriagu** [reply]
Welcome aboard buddy @felixboehm 🍻
